### ISSUE TITLE
Spring 2021 Virtual Meeting

### ISSUE THREAD

**Issue by JanMiksovsky** [WICG/webcomponents#911]
Hey, it's February already! Time to start thinking about when we can meet up again this year to advance the state of web components.

I propose we have a **virtual meeting in the first half of April 2021**.

Agenda TBA, will likely include scoped custom element registries and template parts. Another topic I'd like to discuss: turning the meeting into a regular quarterly meeting so that we can get more consistent progress on challenging topics. We could, for example, meet as a group in January/April/July, then have the fall meeting coincide with TPAC.

If there's interest in an April meeting, I'll set up a Doodle poll.

**Update: The meeting will be Thursday, April 22, 3–7:00 pm UTC.** [Agenda](https://docs.google.com/document/d/160HetuBh6sLArif1y-heWpe749b3KPnX54AS-aB8UQ4/edit#)

**Comment by leobalter** [comment]
> Agenda TBA, will likely include scoped custom element registries and template parts

This would be great! This would be a good time to continue investigating the current polyfill and report feedback and perhaps discuss what is needed to move it forward!

> Another topic I'd like to discuss: turning the meeting into a regular quarterly meeting so that we can get more consistent progress on challenging topics.

For an early feedback on this, I like it! I prefer the dynamics of more often meetings. 4 per year seem fair enough.

**Comment by annevk** [comment]
I'd love for us to go through the backlog and make progress on existing issues.

**Comment by justinfagnani** [comment]
@annevk is there anything we can do to make that more effective this time? Seems like we're always tight for time on that task. Could we nominate issues we'd like to raise?

**Comment by annevk** [comment]
Maybe we should start with it for a change. 😊 And yeah, some pre-triage might help make it more effective. The two links in the README titled "Issues against the DOM and HTML Standard" are particularly relevant.

**Comment by SOULIYA-CHANTHAVONG-MORK** [comment]
mork

**Comment by JanMiksovsky** [comment]
Please vote for the dates you can attend in April: https://doodle.com/poll/aapwby5v8feg33cg

To allow more time for preparation, I've moved the meeting out a bit to the second half of April.

Since the last meeting was amenable to Asia/Pacific workday schedules, this meeting flips again to favoring European schedules. Like last spring's meeting, the proposed meeting time is 3–7 pm UTC.

The Doodle site should show the time in your local time zone, but please confirm the time zone for yourself. All dates fall *after* the US change to Daylight Savings Time and the change to European Summer Time.

**Comment by rniwa** [comment]
I regrettably won't be able to attend the meeting in the proposed time slot due to a medical advisory.

**Comment by justinfagnani** [comment]
@rniwa is there another time period you'd be able to attend?

**Comment by castastrophe** [comment]
I hope you don't mind my contributing to the Doodle.  I would love to attend and at least listen in this time around.  Really interested in participating more as time allows.

**Comment by castastrophe** [comment]
Really excited about both of these topics too: `scoped custom element registries and template parts` 🎉

**Comment by JanMiksovsky** [comment]
I'd like to pick a date for the meeting, but still haven't received many votes for times.

Pinging some people who have attended (or expressed interest in attending) previous meetings @leobalter @hober @bkardell @annevk @cookiecrook @caridy @EisenbergEffect @nicholasrice @travisleithead @pmdartus @diervo @Westbrook 

Ping others as appropriate

**Comment by dbatiste** [comment]
Would it be ok if I attend this virtual meet?

**Comment by EisenbergEffect** [comment]
Would love to attend. Thanks for the heads up @JanMiksovsky !

**Comment by leobalter** [comment]
The [82nd TC39 meeting](https://github.com/tc39/agendas/blob/master/2021/04.md) goes from April 19 to 22 from 10AM to 3PM East Time.

I'd appreciate if we can meet on other non-conflicting days. I might be able to workaround otherwise.

**Comment by leobalter** [comment]
@JanMiksovsky asked me to follow up with the agenda parts.

I'm scheduling this meeting for April 22nd, from 8AM to 12:00PM (noon) Pacific Time.

Here is [the link for the Agenda](https://docs.google.com/document/d/160HetuBh6sLArif1y-heWpe749b3KPnX54AS-aB8UQ4/edit#). It includes the Google Meet and other info. We can add topics to the agenda, and I already included Scoped Custom Elements Registries and Template Parts to kick start. I wonder about other topics that would benefit of this meeting that we can capture in a triage like the suggested by @annevk.

I have set a calendar entry I would be happy to invite more people to track this meeting, you can send me your contact to leobalter at salesforce.

**Comment by Westbrook** [comment]
I’d like to attend the time proposed works for me! Are there specific issues in the APIs we’ll be looking at that I can prepare with? Adobe is quite interested in Scope Registries over the line here!

**Comment by dsadhanala** [comment]
@leobalter thanks for sharing the agenda!
I would like to join as well, interested in the topic `Scoped Custom Element Registries`.

**Comment by bkardell** [comment]
I'm sure we will have a full agenda and this isn't probably the most important item, but I'd be curious for comments from others (besides @rniwa who already commented) on [#6231](https://github.com/whatwg/html/issues/6231) and if there is any need for conversation and opportunity maybe that's interesting?

**Comment by soelen** [comment]
Would love to attend as well

**Comment by btopro** [comment]
Wrote up some thoughts on current proposal as I can't make the proposed time work - https://gist.github.com/btopro/7a10ee782dc2e2a864487e4371baf615
TLDR;
we ship "element registries" currently bc of how we hydrate definitions and wanted to offer a perspective from that perspective. We've had multiple ppl ask us about doing this conceptually with multiple registries / buckets of elements from multiple places.

**Comment by dfreedm** [comment]
I'll be attending!

**Comment by cyns** [comment]
Hello! I've recently joined the Chrome/Chrome OS Accessibility team at Google. I'm planning to attend the meeting to talk about AOM. Looking forward to working with you all.

**Comment by bkardell** [comment]
I think we can close this issue, right?  I

It would be nice if anyone knows where we kept the minutes since it doesn't seem to be here (it's how I wound up here, looking for them) but otherwise ... I don't think there's anything left to say here worth keeping it open.  Sorry I would do it myself if I could :)
